src/App.scss

```diff
@@ -1,3 +1,9 @@
+// Your variable overrides
+$headings-margin-bottom: 1rem;
+
+// Bootstrap and its default variables
+@import '../node_modules/bootstrap/scss/bootstrap';
```

Comment:
We are already importing Bootstrap in index.scss so we don't want to import it again